integration_tests/seeds/sf_product_2_history_data.csv

```diff
@@ -0,0 +1,3 @@
nal_data_source_id,external_id,quantity_unit_of_measure,stock_keeping_unit,features_c,tier_3_price_c,tier_2_price_c,tier_4_price_c,tier_1_price_c,netsuite_conn_sync_in_progress_c,netsuite_conn_celigo_update_c,netsuite_conn_term_contract_pricing_type_c,netsuite_conn_net_suite_id_c,netsuite_conn_net_suite_sync_err_c,netsuite_conn_push_to_net_suite_c,netsuite_conn_item_category_c,netsuite_conn_net_suite_item_type_c,netsuite_conn_sub_type_c,is_new_c,product_metadata_c,product_metadata_del_c,sbqq_asset_amendment_behavior_c,sbqq_asset_conversion_c,sbqq_batch_quantity_c,sbqq_billing_frequency_c,sbqq_billing_type_c,sbqq_block_pricing_field_c,sbqq_charge_type_c,sbqq_component_c,sbqq_compound_discount_rate_c,sbqq_configuration_event_c,sbqq_configuration_field_set_c,sbqq_configuration_fields_c,sbqq_configuration_form_title_c,sbqq_configuration_type_c,sbqq_configuration_validator_c,sbqq_configured_code_pattern_c,sbqq_configured_description_pattern_c,sbqq_cost_editable_c,sbqq_cost_schedule_c,sbqq_custom_configuration_page_c,sbqq_custom_configuration_required_c,sbqq_customer_community_availability_c,sbqq_default_pricing_table_c,sbqq_default_quantity_c,sbqq_description_locked_c,sbqq_discount_category_c,sbqq_discount_schedule_c,sbqq_dynamic_pricing_constraint_c,sbqq_exclude_from_maintenance_c,sbqq_exclude_from_opportunity_c,sbqq_externally_configurable_c,sbqq_generate_contracted_price_c,sbqq_has_configuration_attributes_c,sbqq_has_consumption_schedule_c,sbqq_hidden_c,sbqq_hide_price_in_search_results_c,sbqq_include_in_maintenance_c,sbqq_new_quote_group_c,sbqq_non_discountable_c,sbqq_non_partner_discountable_c,sbqq_option_layout_c,sbqq_option_selection_method_c,sbqq_optional_c,sbqq_price_editable_c,sbqq_pricing_method_c,sbqq_pricing_method_editable_c,sbqq_product_picture_id_c,sbqq_quantity_editable_c,sbqq_quantity_scale_c,sbqq_reconfiguration_disabled_c,sbqq_renewal_product_c,sbqq_sort_order_c,sbqq_specifications_c,sbqq_subscription_base_c,sbqq_subscription_category_c,sbqq_subscription_percent_c,sbqq_subscription_pricing_c,sbqq_subscription_target_c,sbqq_subscription_term_c,sbqq_subscription_type_c,sbqq_tax_code_c,sbqq_taxable_c,sbqq_term_discount_level_c,sbqq_term_discount_schedule_c,sbqq_upgrade_credit_c,sbqq_upgrade_ratio_c,sbqq_upgrade_source_c,sbqq_upgrade_target_c,connector_type_c,pbf_pro_type_discount_c,dimension_c,connector_status_c,dimension_definition_c,ava_sfcpq_tax_code_c,paid_consumption_c,is_complimentary_c,product_external_id_c,blng_billing_rule_c,blng_revenue_recognition_rule_c,blng_tax_rule_c,deployment_date_c,do_not_prorate_c,celigo_sfnsio_netsuite_id_c,sbqq_enable_large_configuration_c,sbqq_pricing_guidance_c,celigo_sfnsio_item_pricing_type_c,celigo_sfnsio_test_mode_record_c,celigo_sfnsio_celigo_last_modified_date_c,celigo_sfnsio_net_suite_record_c,celigo_sfnsio_skip_export_to_net_suite_c,celigo_sfnsio_net_suite_item_type_c,celigo_sfnsio_net_suite_id_c,promo_code_c,product_category_c,product_source_c,non_recurring_c,is_archived,_fivetran_active
```

Comment:
Let's remove this since we are not using it in the end models anymore.

Reply:
Removed